duden/display.py

```diff
@@ -130,46 +130,49 @@ def describe_word(word):
     print(yellow("=" * len(word.title)))
 
     if word.phonetic:
-        print(white(_("Pronunciation:"), bold=True), word.phonetic)
```

Comment:
Please do not remove the `_` function. It is needed for the localization using the `gettext` module.

For example users with Spanish localization see this:

```
LANG=es_ES.UTF-8 duden Kragen
Kragen, der
===========
Tipo de palabra: Substantiv, maskulin
Frecuencia: 2/5
Separación: Kra|gen
Origin: mittelhochdeutsch krage = Hals, Kehle, Nacken; Kragen (1), ursprünglich = Schlund
...
```
(i know the translation is incomplete)

Reply:
sorry my fault. I didn't knew that.